### ISSUE TITLE
non-descriptive error on ambiguous input

### ISSUE THREAD

**Issue by OliverJAsh** [TimBeyer/html-to-vdom#12]
``` js
convertHTML('    <div>foo</div>');
// => TypeError: Cannot convert undefined or null to object
```

**Comment by TimBeyer** [comment]
Still not sure whether that should work or whether you should trim your input ;)

**Comment by TimBeyer** [comment]
I decided that this is not an issue for this library to solve.  
Especially since spaces might also be text nodes with whitespace in them, making this ambiguous input.

You should trim your input yourself if necessary.

**Comment by OliverJAsh** [comment]
Perhaps a friendly error message would be useful?

**Comment by TimBeyer** [comment]
That's actually a good idea. Will implement this.  
I also found an error at the same time, which made it impossible to convert text-only input. This actually caused the bug you saw.

So I will combine both, and make it possible to convert text only, but throw a descriptive error in case multiple tags are passed in as the root, like `convertHTML('<div></div><span></span>');`.

**Comment by TimBeyer** [comment]
Published as [v0.5.4](https://github.com/TimBeyer/html-to-vdom/releases/tag/v0.5.4)
